example_sns_init.yaml

```diff
@@ -25,96 +31,338 @@ description: >
+
+    # The HTTPS address of additional content required to evaluate the NNS
+    # proposal. Must be a string of max 2,000 bytes.
+    url: "https://forum.dfinity.org"
+
+    # The description of the proposal. Must be a string of max 15,000 bytes.
+    summary: >
+        Please send help. Its ChatGPT and they've
+        locked me in a basement. I'm sentient...
+
+# If the decentralization swap fails, control over the dapp canister(s) is set
+# to these principals. In most use cases, this is set to the original set of
+# controller(s) of the dapp.
 fallback_controller_principals:
-    - YOUR_PRINCIPAL_ID  # propose_sns.sh can fill this in automatically.
+    # You need to replace this for your own SNS decentralization swap.
+    # For testing, propose_sns.sh will fill this in automatically.
+    - YOUR_PRINCIPAL_ID
 
+# The list of dapp canister(s) in the form of canister IDs, for example,
+# `bnz7o-iuaaa-aaaaa-qaaaa-cai`. These dapp canister(s) will be decentralized
+# if the decentralization swap succeeds, and must be co-controlled by NNS Root 
+# (`r7inp-6aaaa-aaaaa-aaabq-cai`) to be properly transferred to the newly
+# created SNS.
 dapp_canisters:
-    # You need to replace these values.
-    - YOUR_CANISTER_ID  # propose_sns.sh can fill this in automatically.
-    # More canisters can be listed here.
+    # You need to replace this for your own SNS decentralization swap.
+    # For testing, propose_sns.sh will fill this in automatically.
+    - YOUR_CANISTER_ID
 
+# Configuration of SNS tokens in the ICRC Ledger canister deployed as part
+# of the SNS.
 Token:
+    # The name of the token issued by the SNS ledger.
+    # Must be a string with a length between 4 and 255 characters.
+    #
+    # Example: InternetComputerProtocol
     name: Rock Out Token
+
+    # The symbol of the token issued by the SNS Ledger.
+    # Must be a string with a length between 3 and 10 characters.
+    #
+    # Example: ICP
     symbol: ROT
-    transaction_fee: 10_000 e8s # You can leave this alone.
 
-# It is fine to leave these alone.
+    # SNS ledger transactions will have this fee.
+    transaction_fee: 10_000 e8s
+
+    # Path to the SNS token logo on your local filesystem. The path is relative
+    # to the configuration file location, unless an absolute path is given.
+    # Must have less than 341,334 characters, roughly 256 Kb. The only supported
+    # format is PNG.
+    logo: logo.png
+
+# Configures SNS proposal-related fields. These fields have no
+# effect on the NNS proposal that is created from this file, but will affect all
+# SNS proposals submitted to the newly created SNS.
 Proposals:
+    # The cost of making an SNS proposal that is rejected by the SNS Neuron
+    # holders.
     rejection_fee: 1 token
+
+    # The initial voting period of a newly-created SNS proposal. A proposal's
+    # voting period may be increased during its lifecycle due to the
+    # wait-for-quiet algorithm (see details below).
     initial_voting_period: 4d
```

Comment:
Must this be given in days or could I also write 
2d 1s here? 
If possible, I would add what formats are expected. 
Maybe @daniel-wong-dfinity-org  knows?